pkg/auth/auth.go

```diff
@@ -465,6 +465,9 @@ func (a *Authenticator) SetCSRFCookie(path string, w *http.ResponseWriter) {
 		Path:     path,
 		Secure:   a.secureCookies,
 	}
+	if a.secureCookies {
+		cookie.SameSite = 4
```

Comment:
Sorry, I believe my comment in the BZ was wrong. We probably always want `Lax` reading about this more closely.

```suggestion
		cookie.SameSite = cookie.SameSiteLaxMode
```

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite#lax

Reply:
Makes sense 👍